### PR TITLE
ASB june

### PR DESCRIPTION
**Pull request by SaliniVenate**

### CODE REVIEW

File: aosp_diff/preliminary/frameworks/av/07_0007-Fix-UAF-in-clearkey-service-s-MemoryFileSystem.patch [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
++    Mutex::Autolock lock(mFileHandleLock);
+     if (mFileHandle.DeleteLicense(licenseName)) {
+         return Status::OK;
+     }
+@@ -733,6 +741,8 @@ Return<Status> DrmPlugin::removeOfflineLicense(const KeySetId& keySetId) {
+
+ Return<void> DrmPlugin::getOfflineLicenseState(const KeySetId& keySetId,
+         getOfflineLicenseState_cb _hidl_cb) {
++    Mutex::Autolock lock(mFileHandleLock);
++
```
Comment (tanujtekriwal) [inline aosp_diff/preliminary/frameworks/av/07_0007-Fix-UAF-in-clearkey-service-s-MemoryFileSystem.patch:107]: extra line.

File: aosp_diff/preliminary/frameworks/base/23_0023-Limit-maximum-allowed-size-for-a-status-bar-icon.patch [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
++import android.graphics.Bitmap;
+ import android.graphics.Color;
+ import android.graphics.drawable.Icon;
+ import android.os.UserHandle;
+@@ -123,4 +124,13 @@ public class StatusBarIconViewTest extends SysuiTestCase {
+         assertEquals("Transparent backgrounds should fallback to drawable color",
+                 color, mIconView.getStaticDrawableColor());
+     }
+-}
+\ No newline at end of file
```
Comment (tanujtekriwal) [inline aosp_diff/preliminary/frameworks/base/23_0023-Limit-maximum-allowed-size-for-a-status-bar-icon.patch:64]: See this. If this is correct.

File: aosp_diff/preliminary/packages/apps/Settings/03_0003-Fix-Settings-WifiScanModeActivity-Overlaid-issue.patch [hunk 3]
```diff
@@ -0,0 +1,44 @@
+From 3ce3f0a163ead5e59fe132244fd1cfe15171bc16 Mon Sep 17 00:00:00 2001
+From: Weng Su <wengsu@google.com>
+Date: Fri, 29 Jan 2021 17:24:51 +0800
+Subject: [PATCH] Fix Settings WifiScanModeActivity Overlaid issue
+
+- Disallowed non system overlay windows
```
Comment (tanujtekriwal) [inline aosp_diff/preliminary/packages/apps/Settings/03_0003-Fix-Settings-WifiScanModeActivity-Overlaid-issue.patch:6]: These 2 lines removed from comments?

**Review by tanujtekriwal — COMMENTED** [review]
Please check my 3 comments.

File: aosp_diff/preliminary/packages/apps/Settings/03_0003-Fix-Settings-WifiScanModeActivity-Overlaid-issue.patch (hunk 3 above)
Comment (SaliniVenate) [reply aosp_diff/preliminary/packages/apps/Settings/03_0003-Fix-Settings-WifiScanModeActivity-Overlaid-issue.patch:6]: Those 2 lines are part of commit message:

" From 37231e5e6d4df4035bbcdca744397bee8f6fbbf9 Mon Sep 17 00:00:00 2001
From: Weng Su <wengsu@google.com>
Date: Fri, 29 Jan 2021 17:24:51 +0800
Subject: [PATCH] [Security Report] Fix Settings WifiScanModeActivity Overlaid
 issue

- Disallowed non system overlay windows

- Screenshot
  https://screenshot.googleplex.com/77fJ9QN6pV4zFYc

Bug: 174047492
Test: manual test"

File: aosp_diff/preliminary/frameworks/av/07_0007-Fix-UAF-in-clearkey-service-s-MemoryFileSystem.patch (hunk 1 above)
Comment (SaliniVenate) [reply aosp_diff/preliminary/frameworks/av/07_0007-Fix-UAF-in-clearkey-service-s-MemoryFileSystem.patch:107]: > extra line.

This extra line is present in Google patch also.

File: aosp_diff/preliminary/frameworks/base/23_0023-Limit-maximum-allowed-size-for-a-status-bar-icon.patch (hunk 2 above)
Comment (SaliniVenate) [reply aosp_diff/preliminary/frameworks/base/23_0023-Limit-maximum-allowed-size-for-a-status-bar-icon.patch:64]: > See this. If this is correct.

Double checked and it looks fine. This was introduced by git format-patch.

**Review by shyjumon-n — APPROVED** [review]
Google security patches for ASB-June.
Looks Good To Me.

**Pull request merged**